src/OpenConext/ProfileBundle/Service/InformationRequestMailService.php

```diff
@@ -76,21 +76,13 @@ public function sendInformationRequestMail()
     {
         $user = $this->userService->getUser();
 
-        $attributes = $this->attributeFilter->filter($user->getAttributes());
-
-        $body = $this->templateEngine->render(
-            '@OpenConextProfile/InformationRequest/email.html.twig',
-            ['attributes' => $attributes]
-        );
-
-        /** @var Message $message */
-        $message = $this->mailer->createMessage();
-        $message
-            ->setFrom($this->mailFrom->getEmailAddress())
-            ->setTo($this->mailTo->getEmailAddress())
-            ->setSubject(sprintf('Information request for user: %s', $user->getId()))
-            ->setBody($body, 'text/html', 'utf-8');
-
-        $this->mailer->send($message);
+        /** @var Email $message */
+        $email = (new TemplatedEmail())
+            ->from($this->mailFrom->getEmailAddress())
+            ->to($this->mailTo->getEmailAddress())
+            ->subject(sprintf('Personal debug info of %s', $user->getId()))
+            ->htmlTemplate('@OpenConextProfile/InformationRequest/email.html.twig')
+            ->context(['attributes' => $user->getAttributes()]);
```

Comment:
These are the wrong attributes. Previously we used: 

```
$attributes = $this->attributeFilter->filter($user->getAttributes());
```

And you changed it to:

```
$user->getAttributes()
```